src/utils/openwhisk.js

```diff
@@ -132,8 +132,8 @@ function extractActionContext(params) {
     secrets,
     request: {
       params: disclosed,
-      headers: __ow_headers,
-      method: __ow_method,
+      headers: __ow_headers || {},
```

Comment:
I wonder if it would make sense to populate some default headers, too. e.g. `User-Agent: OpenWhisk`, `X-CDN-Request-Id: manual`